velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -223,6 +223,10 @@ class VectorFuzzer {
   TypePtr randType(int maxDepth = 5);
   RowTypePtr randRowType(int maxDepth = 5);
 
+  // Generate a random non-floating-point primitive type to be used as group key
+  // for aggregations.
+  TypePtr randGroupKeyType();
```

Comment:
typos: "group key" -> "group-by key" or "grouping key". This applies to other keys as well, e.g. join key. Maybe, rename to avoid specifying the usage of the key: randScalarNonFloatingPointType.